recipes/or-tools/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -BUILD_DEPS=OFF \
+      -DUSE_SCIP=OFF \
+      -S. \
+      -Bbuild
```

Comment:
last comment: here you are building (and installing) all C++ samples, not sure it is usefull from a distro package perspective.
You could disable them using: `-DBUILD_SAMPLES=OFF -DBUILD_EXAMPLES=OFF`
note: currently there is no option to disable the install i.e. only build them and use them as test but not install them
note2: seems to currently only impact C++ samples

src:
https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/CMakeLists.txt#L115
and https://github.com/google/or-tools/blob/b37d9c786b69128f3505f15beca09e89bf078a89/CMakeLists.txt#L127
c++ sample cmake function:
https://github.com/google/or-tools/blob/abbeaace99f8aa47d74a294a30dc4fa7a52e991f/cmake/cpp.cmake#L303

Reply:
Thanks for the heads up, yeah probably something that we should remove. Was just thinking about that in terms of testing.

I'm a bit busy the next week but will try to remove those, add the tests and add a python package afterwards so that we have an MVP we can publish and then go from there RE other languages.